src/app/pages/private/shop-approval/shop-approval.component.html

```diff
@@ -0,0 +1,243 @@
+          <div class="p-3">
+            <label class="text-primary">Icon</label>
+            <div class="row pl-3">
+              <div *ngIf="data['icon'];else iconNotFound" class="ml-3">
+                <img [src]="data['icon']" height="70" width="70"/>
+              </div>
+              <ng-template #iconNotFound>
+                <span class="small text-light-gray">No icon found</span>
+              </ng-template>
+            </div>
+          </div>
+          <div class="p-3">
+            <label class="text-primary">Cover Photos</label>
+            <div class="row pl-3">
+              <div *ngFor="let imgSrc of data['cover_photos'];">
+                <div class="image-preview ml-3 pt-1">
+                  <img [src]="imgSrc['url']" height="34" width="70"/>
+                </div>
+              </div>
+              <div *ngIf="data['cover_photos'].length==0">
+                <span class="small text-light-gray">No cover photos found</span>
+              </div>
+            </div>
+          </div>
+        </ng-template>
+      </li>
+      <li ngbNavItem="payment-tab">
+        <a ngbNavLink>Payment</a>
+        <ng-template ngbNavContent>
+          <div class="row p-3">
+            <div class="col-md-3 col-lg-3">
+              <label class="text-primary">Account Number</label>
+              <br>
+              <span>{{ data['payment']['account_number'] }}</span>
+            </div>
+            <div class="col-md-3 col-lg-3">
+              <label class="text-primary">IFSC Code</label>
+              <br>
+              <span>{{ data['payment']['account_ifsc'] }}</span>
+            </div>
+            <div class="col-md-3 col-lg-3">
+              <label class="text-primary">Account Holder Name</label>
+              <br>
+              <span>{{ data['payment']['account_holder'] }}</span>
+            </div>
+          </div>
+          <div class="row p-3">
+            <div class="col-md-3 col-lg-3">
+              <label class="text-primary">PAN Number</label>
+              <br>
+              <span>{{ data['payment']['pan'] }}</span>
+            </div>
+            <div class="col-md-3 col-lg-3">
+              <label class="text-primary">GST Number</label>
+              <br>
+              <span>{{ data['payment']['gst'] }}</span>
+            </div>
+          </div>
+        </ng-template>
+      </li>
+    </ul>
+    <div [ngbNavOutlet]="nav"></div>
+  </div>
+</div>
+
+<div class="mt-4">
+  <ngb-accordion #accordion="ngbAccordion" [destroyOnHide]="false" activeIds="ngb-panel-0">
+    <ngb-panel id="updateStatusPanel">
+      <ng-template class="panel-header" let-opened="opened" ngbPanelHeader>
+        <accordion-header (click)="expandPanel(accordion, 'updateStatusPanel')" [opened]="opened"
+                          title='Update Status'></accordion-header>
+      </ng-template>
+      <ng-template ngbPanelContent>
+        <div class="row">
+          <div *ngIf="isTerminalStatus" class="col-3 pt-2 pl-4">
```

Comment:
Can we try NgSwitch? https://angular.io/api/common/NgSwitch
Also isTerminalStatus method may not be required